stats/opentelemetry/e2e_test.go

```diff
@@ -14,16 +14,19 @@
  * limitations under the License.
  */
 
-package opentelemetry_test
+package opentelemetry
```

Comment:
why change this?

Reply:
Ohh this was done while updating the go.mod files. Reverted it back, thanks for noticing.